packages/EasyStandard/tests/Rector/RestoreDefaultNullToNullableTypeParameterRector/Fixture/SecondParameterMissingDefaultValue.php.inc

```diff
@@ -0,0 +1,24 @@
+<?php
+
+class SecondParameterMissingDefaultValue
+{
+    public function __construct(string $param, ?string $value)
```

Comment:
Let's use `private function someMethod(string $param, ?string $value)` here instead of the constructor? To not concentrate tests on one method "type".

Reply:
Done.